panther_utils/include/panther_utils/moving_average.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef PANTHER_UTILS_MOVING_AVERAGE_HPP_
```

Comment:
There could be a reset function to make it more general. The reset might set the whole internal queue to some value rather than zero. That will also require additional test case